tutorials/tutorial_views.py

```diff
@@ -0,0 +1,944 @@
+"""
+    mss.tutorials.tutorial_views
+    ~~~~~~~~~~~~~~~~~~~
+
+    This python script generates an automatic demonstration of how to use the top view, side view, table view and
+    linear view section of Mission Support System in creating a project and planning the flightrack.
+
+    This file is part of mss.
+
+    :copyright: Copyright 2021 Hrithik Kumar Verma
+    :copyright: Copyright 2021 by the mss team, see AUTHORS.
+    :license: APACHE-2.0, see LICENSE for details.
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+"""
+
+import pyautogui as pag
+import multiprocessing
+import sys
+from sys import platform
```

Comment:
Oh, It was by mistake. I will remove that `import sys`

Reply:
I think I would unify the usage. you could do 

from sys import platform, exit

or use sys.platform 

I would use sys.platform because it is already imported and I am used to read sys.exit()